sdks/python/tox.ini

```diff
@@ -33,6 +33,16 @@ extras = test
 whitelist_externals =
   find
   time
+deps =
+  grpcio-tools==1.3.5
+  cython: cython==0.26.1
+
+# These 2 magic command overrides are required for Jenkins builds.
+# Otherwise we get "OSError: [Errno 2] No such file or directory" errors.
+# Source:
+# https://github.com/tox-dev/tox/issues/123#issuecomment-284714629
+install_command = {envbindir}/python {envbindir}/pip install {opts} {packages}
```

Comment:
Do you need the `{envbindir}/python` part? `pip` might already be an executable.

Reply:
Yes, I tried without it and it fails.